routes/api.php

```diff
@@ -13,68 +13,39 @@
 |
 */
 
-// Route::middleware('auth:api')->get('/user', function (Request $request) {
-//     return $request->user();
-// });
-
-Route::get('/homepage_data', function () {
+Route::get('/homepage_data', function () { // Used from DeviceController, tested.
     return App\Http\Controllers\ApiController::homepage_data();
 });
 
-Route::get('/party/{id}/stats', function ($id) {
+Route::get('/party/{id}/stats', function ($id) { // Not used but worth keeping, tested.
```

Comment:
This is used from trp.org
(/var/www/therestartproject.test/wp-content/themes/restart/js/load-specific/fixometer-stats.js)

Reply:
Ok - added a comment.